src/app/tests/suites/TestColorControl_9_1.yaml

```diff
@@ -0,0 +1,770 @@
+# Copyright (c) 2023 Project CHIP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Copy/ Paste of the TC_CC_9.1 with shorter delays for Github CI only
+name: 25.3.24. [TC-CC-9.1] ColorLoopSet Validation
+
+config:
+    nodeId: 0x12344321
+    timeout: 400
+    cluster: "Color Control"
+    endpoint: 1
+
+tests:
+    - label: "Wait for the commissioned device to be retrieved"
+      cluster: "DelayCommands"
+      command: "WaitForCommissionee"
+      arguments:
+          values:
+              - name: "nodeId"
+                value: nodeId
+
+    - label: "TH writes 0 to the Options attribute"
+      command: "writeAttribute"
+      attribute: "Options"
+      arguments:
+          value: 0
+
+    - label: "TH sends On command to DUT"
+      cluster: "On/Off"
+      command: "On"
+
+    - label:
+          "TH sends EnhancedMoveToHue command to DUT with EnhancedHue=0x4000,
+          Direction=0x00 (shortest distance) and TransitionTime=0 (immediately)."
+      command: "EnhancedMoveToHue"
+      arguments:
+          values:
+              - name: "EnhancedHue"
+                value: 16384
+              - name: "Direction"
+                value: 0
+              - name: "TransitionTime"
+                value: 0
+              - name: "OptionsMask"
+                value: 0
+              - name: "OptionsOverride"
+                value: 0
+
+    - label: "Wait for 1000ms"
```

Comment:
The label here does not match the actual delay.

Reply:
done